front-end/studio/src/app/pages/apis/{apiId}/editor/editor.component.ts

```diff
@@ -193,14 +193,24 @@ export class ApiEditorComponent implements OnChanges, OnInit, OnDestroy, IEditor
      */
     public onGlobalKeyDown(event: KeyboardEvent): void {
         // TODO skip any event that was sent to an input field (e.g. input, textarea, etc)
-        if (event.ctrlKey && event.key === 'z' && !event.metaKey && !event.altKey) {
+        if (ApiEditorComponent.isUndo(event)) {
             this.undoLastCommand();
         }
-        if (event.ctrlKey && event.key === 'y' && !event.metaKey && !event.altKey) {
+        if (ApiEditorComponent.isRedo(event)) {
             this.redoLastCommand();
         }
     }
 
+    private static isUndo(event: KeyboardEvent): boolean {
+        return (event.ctrlKey && event.key === 'z' && !event.metaKey && !event.altKey) ||
+            (event.metaKey && event.key === 'z' && !event.ctrlKey && !event.altKey);  // macOS: cmd + z
```

Comment:
My thoughts on this was that we support both options (macOS shortcuts and windows/linux). So you can do it the Linux/Windows way on Mac if you want.